src/aws/guards/roles.guard.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
```

Comment:
Wait, we already have role based auth from JPAL. Is this from there? Or somewhere else?

Reply:
Hmmm I'm a little confused, wdym? The RolesGuard is based off of what JPAL does, as well as the tutorial from Nest about how to do this, but JPAL stores their users and roles in a database. We opted to use Cognitp groups to track roles instead, so it's a little different. 

We didn't have role-based authentication prior to this in Breaktime...

Reply:
ack. that makes sense!